src/ORM/Connect/Database.php

```diff
@@ -20,6 +20,23 @@
 abstract class Database
 {
 
+    const PARTIAL_QUERY = 'partial_query';
+    const FULL_QUERY = 'full_query';
+
+    /**
+     * To use, call from _config.php
+     * Example:
+     * <code>
+     * Database::setWhitelistQueryArray([
+     *      'Qualmark' => 'partial_query',
+     *      'SELECT "Version" FROM "SiteTree_Live" WHERE "ID" = ?' => 'full_query',
+     * ])
+     * </code>
+     * @internal
```

Comment:
Carrying on from the previous discussion, I’m sorry but I plainly don’t agree with the previous feedback. We have a lot of protected properties with public accessors already that are not marked as internal, and I think using it here is not a strong enough use case to justify it since it should be used sparingly.

Can you please remove the annotation?

Reply:
In terms of consistency, I agree that I should remove `@internal` if it's not the same all over the place but I still would leave the rest of the docs with example provided.

Reply:
Yes please